person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def create_rental(date, book)
+    Rental.new(date, self, book)
+  end
+
```

Comment:
- Good job done so far, but the parameters for your Rental class were passed as `date`, `book`, `person` in that order, please fix the parameter arrangement to avoid your code running into errors 👍 👍 
```ruby
 Rental.new(date, book, self)
```